cmd/monaco/integrationtest/v2/config_restore_e2e_test.go

```diff
@@ -253,33 +263,23 @@ func preparation_uploadConfigs(t *testing.T, fs afero.Fs, suffixTest string, con
 	return suffix, nil
 }
 
-func execution_downloadConfigsWithCLIParameters(t *testing.T, fs afero.Fs, downloadFolder string, _ string,
-	apisToDownload string, settingsToDownload string) error {
+func execution_downloadConfigsWithCLIParameters(
+	t *testing.T,
+	fs afero.Fs,
+	downloadFolder string,
+	_ string,
+	apisToDownload string,
+	settingsToDownload string,
+	oauth bool,
+) error {
 	log.Info("BEGIN DOWNLOAD PROCESS")
 
 	downloadFolder, err := filepath.Abs(downloadFolder)
 	if err != nil {
 		return err
 	}
-	var parameters []string
-	if apisToDownload == "all" {
-		parameters = []string{
-			"download",
-			"direct",
-			os.Getenv("URL_ENVIRONMENT_1"),
-			"TOKEN_ENVIRONMENT_1",
-			"--verbose",
-			"--output-folder", downloadFolder,
-		}
-	} else {
-		parameters = []string{
-			"download",
-			"direct",
-			os.Getenv("URL_ENVIRONMENT_1"),
-			"TOKEN_ENVIRONMENT_1",
-			"--verbose",
-			"--output-folder", downloadFolder,
-		}
+	parameters := []string{"download", "--verbose", "--output-folder", downloadFolder}
+	if apisToDownload != "all" {
```

Comment:
nit: could drop the nested if and just do 
```suggestion
	if apisToDownload != "all" && apisToDownload != "" {
```

Reply:
there is 2 nested if branching here...